about.html

```diff
@@ -1,64 +1,108 @@
-<!doctype html>
+<!DOCTYPE html>
 
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-
-  <title>Sprint Challenge - About</title>
-
-  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
-
-</head>
-
-<body>
+  <head>
+    <meta charset="utf-8" />
+
+    <title>Sprint Challenge - About</title>
+
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto|Rubik"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/index.css" />
+  </head>
+  <header>
+    <img src="img/lambda-black.png" alt="Lambda logo" />
+    <nav id="nav-bar">
+      <a href="index.html">Home</a>
+      <a href="about.html">About</a>
```

Comment:
One thing you can do here instead of href=" " is do href="#" this is a placeholder and just reloads the page.